### PR TITLE
test added, menus collapsed, packages structure

### PR DESCRIPTION
**Pull request by andresin87**

### CODE REVIEW

File: packages/components/button/package.json [hunk 1]
```diff
@@ -1,15 +1,22 @@
 {
   "name": "@ds-name/button",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "Button component is used to trigger an action or event, such as submitting a form, opening a Dialog, canceling an action, or performing a delete operation.",
   "type": "module",
   "main": "dist/cjs/index.js",
+  "types": "./dist/main.d.ts",
```
Comment (andresin87) [inline packages/components/button/package.json:7]: we need to find the way to generate the types, and provide them as is

Comment (andresin87) [inline packages/components/button/package.json:7]:
```suggestion
  "types": "./dist/[cjs | esm]main.d.ts",
```

Comment (Powerplex) [reply packages/components/button/package.json:7]: Maybe this can help: https://antfu.me/posts/publish-esm-and-cjs#tsup

File: package.json [hunk 2]
```diff
@@ -19,10 +20,9 @@
     "typecheck": "tsc",
     "write-themes": "ts-node-esm writeThemes.ts",
     "write-tailwind-conf": "ts-node-esm writeTailwindConfig.ts",
-    "postinstall": "husky install"
```
Comment (acd02) [inline package.json:22]: Why remove this line?

File: package.json [hunk 3]
```diff
@@ -19,10 +20,9 @@
...
+    "pre-push": "npm run test"
```
Comment (acd02) [inline package.json:23]: Maybe move this inside `.husky/pre-push`

**Pull request closed without merging**